### ISSUE TITLE
Deprecated cartopy calls: .xlabels_top & .ylabels_right

### ISSUE THREAD

**Issue by srmullens** [tropycal/tropycal#131]
A warning from cartopy/mpl/gridliner.py says that .xlabels_top and .ylabels_right is deprecated. 

```
.../python3.8/site-packages/cartopy/mpl/gridliner.py:324: 
UserWarning: The .xlabels_top attribute is deprecated. Please use .top_labels to toggle visibility instead.

.../python3.8/site-packages/cartopy/mpl/gridliner.py:360: 
UserWarning: The .ylabels_right attribute is deprecated. Please use .right_labels to toggle visibility instead.
```

The offending lines are in tropycal / plot / plot.py lines 232, 233, 242, 243. Below are the lines around 232 and 233.

```
#Second call with labels but no gridlines
gl = self.ax.gridlines(...)
            
gl.xlabels_top = False          # <-- Offending line
gl.ylabels_right = False        # <-- Offending line
gl.xlocator = mticker.FixedLocator(meridians2)
gl.ylocator = mticker.FixedLocator(parallels)
gl.xformatter = LONGITUDE_FORMATTER
gl.yformatter = LATITUDE_FORMATTER
```

Solution: Replace the offending lines with:
```
gl.top_labels = False
gl.right_labels = False
```

**Comment by tomerburg** [comment]
What version of Cartopy are you using? I've done my testing thus far with v0.17 as I've run into problems with using v0.18 or newer, so if it's the latter I'll create a new cartopy v0.18 testing environment. Can you confirm on your end in the meantime that the new lines you suggested eliminate the deprecation warning and still plot as intended?

**Comment by tomerburg** [comment]
Now that I've finally had time to look into this, I have been unable to set up an environment with cartopy >=0.18 (an issue involving the server I use), but testing it with older versions of cartopy the syntax you suggested doesn't work, suggesting that if there was any overlap between the old and new syntax before the former was deprecated, it was most likely brief.

In the meantime I replaced the offending lines with a try-catch block, trying the old syntax, and assuming that the code crashes on newer cartopy versions, it then switches to the new syntax. This is now up on the [fix-bugs](https://github.com/tropycal/tropycal/tree/fix-bugs) branch - can you test to make sure it works on your end by plotting any storm (the top & right lat/lon labels should not be visible if this works correctly)?

**Comment by srmullens** [comment]
I made these edits within my own code. The deprecation warning still shows up. But, the plot is indeed created without the try block failing and going to the except block.

With the admission that I don't have a detailed knowledge of how the try-except logic works with warnings, perhaps because the code DOES still work the try block doesn't fail, despite the warning. So this logic is probably fine for future versions of Cartopy where they remove the deprecation warning, triggering the failure of the try clause. 

If the above is true, could consider switching the pairs of lines between the try and except clause? I think that would make the deprecation warning not show up either way. Ultimately, if the plot works, then perhaps it doesn't matter.

**Comment by tomerburg** [comment]
This bug fix was implemented with version v0.3.2, so I’ll be closing this issue. Thanks again for reporting this!
